modules/account-lib/src/coin/sol/transferBuilder.ts

```diff
@@ -12,6 +12,8 @@ import assert from 'assert';
 export interface SendParams {
   address: string;
   amount: string;
+  mint?: string;
+  source?: string;
```

Comment:
not obvious what `source` is for - could you add a comment to explain what this is used for? and why it needs to be different per recipient address?